jenkins/pull_request_creator.py

```diff
@@ -100,6 +101,8 @@ def _create_new_pull_request(self):
         LOGGER.info("Created PR: https://github.com/{}/pull/{}".format(
             self.repository.full_name, pr.number
         ))
+        if self.output_pr_url_for_github_action:
+            print(f'::set-output name=generated_pr::https://github.com/{self.repository.full_name}/pull/{pr.number}')
```

Comment:
Yes and no. Github actions won't parse this line as potentially a command for how it should behave if the line begins with the stuff that all logger-output logs are output with. I can possibly create a separate logger for this one instance and configure that logger not to prepend anything, but this is a lot simpler. I can add a comment though, since it piqued your curiosity and likely will others' as well.